evmcore/tx_validation.go

```diff
@@ -28,9 +28,9 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
-// validationOptions is a set of options to adjust the validation of transactions
-// according to the current state of the transaction pool.
-type validationOptions struct {
+// networkOptions defines the set of active network rules and parameters
+// used to validate transactions based on the current fork state.
+type networkOptions struct {
```

Comment:
nit: these are not options for any network ... so the name is not very helpful. Consider including the fact that these are still options for the transaction validation into the type name.